app/controllers/spree/orders_controller_decorator.rb

```diff
@@ -165,6 +165,12 @@ def order_cycle_expired
 
   private
 
+  def ensure_distributor
```

Comment:
Oops no - I added the method to the wrong [non admin] orders controller and forgot to remove it.